reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -513,7 +514,7 @@ class StaticHtmlReporter : Reporter {
                         dd {
                             row.declaredLicenses.forEach {
                                 div {
-                                    +if (it.contains(",")) "\"$it\"" else it
```

Comment:
Was eliminating the quotation logic already possible before this commit? If so consider doing it in a separate commit before this one.

Reply:
No, before `it` could contain commans, now this is not possible anymore.